C/common/include/string_utils.h

```diff
@@ -28,7 +28,9 @@ string StringSlashFix(const string& stringToFix);
 std::string evaluateParentPath(const std::string& path, char separator);
 std::string extractLastLevel(const std::string& path, char separator);
 
-void StringStripCRLF(std::string& StringToManage);
+void   StringStripCRLF(std::string& StringToManage);
```

Comment:
What might be useful would be removing duplicate white space, i.e. a run of 4 spaces becomes 1. That way at least some spacing is preserved.